ste/xferLogPolicy.go

```diff
@@ -98,8 +98,8 @@ func NewRequestLogPolicyFactory(o RequestLogOptions) pipeline.Factory {
 				}
 
 				pipeline.WriteRequestWithResponse(b, prepareRequestForLogging(request), response.Response(), err)
-				if logLevel <= pipeline.LogError {
-					b.Write(stack()) // For errors (or lower levels), we append the stack trace (an expensive operation)
+				if logLevel <= pipeline.LogError && !httpError {
```

Comment:
Can you whack a brief comment in here, to explain why we are not logging call stacks for HTTP errors. Namely they don't add much value (if any) but they do add a lot of bulk to the log in cases such as lots of 503 responses.

Reply:
Sure thing!